### PR TITLE
Luke

### PR DESCRIPTION
**Pull request by u19010240**

### CODE REVIEW

**Comment by EllenLyne** [comment]
Hey Luke, I will review when it's not a "work in progress" anymore

File: Ionic/src/app/pages/booking/booking-type/add-btype/add-btype.component.html [hunk 1]
```diff
@@ -29,6 +29,40 @@
...
+      <br/>
+    </ion-item>
+    <span class="error ion-padding" *ngIf="errorControl.bookingTypePrice.errors?.required && cBookingTypeForm.get('bookingTypePrice').touched">
+     Price is required <br/>
+    </span>
+    <span class="error ion-padding" *ngIf="errorControl.bookingTypePrice.errors?.min">
+     Price must be greater than 0 <br/>
+    </span>
+    <ion-item>
+      <ion-label slot="start" id="clrSelector">Colour: </ion-label>
```
Comment (EllenLyne) [inline Ionic/src/app/pages/booking/booking-type/add-btype/add-btype.component.html:45]: Hey Luke, just a suggestion maybe call it "Pick a color" or something to make it a bit more user friendly

File: Ionic/src/app/pages/booking/booking-type/confirm-btype/confirm-btype.component.html [hunk 2]
```diff
@@ -21,6 +21,22 @@
     <ion-label class="ion-text-left">Description:</ion-label>
     <ion-label class="ion-text-wrap ion-text-right">{{bookingType.description}}</ion-label>
   </ion-item>
+  <ion-item >
+    <ion-label class="ion-text-left">Colour:</ion-label>
+    <input disabled type="color" value="{{bookingType.colour}}" required="" id="bookingTypeColour" slot="end"/>
```
Comment (EllenLyne) [inline Ionic/src/app/pages/booking/booking-type/confirm-btype/confirm-btype.component.html:26]: Same as previous comment

File: Ionic/src/app/pages/venue/confirm-venue/confirm-venue.component.ts [hunk 3]
```diff
@@ -57,7 +73,7 @@ export class ConfirmVenueComponent {
       }
           //dismiss modal
           this.global.dismissModal();
-    }); 
+    });
 
   }
```
Comment (EllenLyne) [inline Ionic/src/app/pages/venue/confirm-venue/confirm-venue.component.ts:79]: I am glad you implemented the new duplicate code into venue, saves some time in future.. thanks

**Review by EllenLyne — APPROVED** [review]
Everything looks good, 
I recommend we change the "colour" attribute name to "Pick a colour" but it' only my preference,

**Review by shannonlnoel — APPROVED** [review]
Nice <3

**Review by u19010240 — COMMENTED** [review]
Great job Luke

**Pull request merged**